readthedocs/core/static-src/core/js/doc-embed/sponsorship.js

```diff
@@ -149,6 +149,7 @@ function Promo(data) {
     this.div_id = data.div_id || '';
     this.html = data.html || '';
     this.display_type = data.display_type || '';
+    this.view_tracking_url = data.view_url;
```

Comment:
Are we already sending this field in our data? Just asking because I don't see any Python code modified in this PR.

Reply:
You need to look at the related -ext PR https://github.com/rtfd/readthedocs-ext/pull/200